README.md

```diff
@@ -1,210 +1,474 @@
+
+- **Synthetic spectrum generation**  
+  Generate spectral distributions from analytical models:
+  - **Planck’s law** for blackbody radiators  
+  - **Gaussian functions** for custom filter shapes  
+  - **RGB channel mixtures** to approximate display pixel spectra
+
+- **Multiple CIE standard observers**  
+  Compute tristimulus values using any supported observer:
+  - **CIE 1931** 2° Standard Observer  
+  - **CIE 1964** 10° Standard Observer  
+  - **CIE 2015** Cone Fundamental based Standard Observers (2° & 10°)
+
+- **Color appearance and uniform color spaces**  
+  Convert spectral data into perceptually uniform models and appearance spaces such as **CIELAB**, **CIELUV**, **CIECAM02**, and **CAM16**, with support for many ΔE color-difference metrics.
+
+- **Advanced colorimetry support**  
+  With physiological cone fundamentals–based observers and modern interpolation methods, this library overcomes the limitations of the original CIE 1931 model and is suitable for high-precision LED, display, and imaging applications.
+
+## Installation
 
 To use this library in a Rust application, run the command:
  ```bash
     cargo add colorimetry
 ```
 or add this line to the dependencies in your Cargo.toml file:
 ```toml
-    colorimetry = "0.0.3"
+    colorimetry = "0.0.4"
 ```
-The easiest way to use the objects and functions library is through its prelude.
-This example calculates the chromaticity values of the CIE D65 illuminant.
+The easiest way to use the objects and functions in this library is through its prelude.
+
 ```rust
     use colorimetry::prelude::*;
-    let xy = CIE1931.xyz(&D65, None).chromaticity();
-    approx::assert_abs_diff_eq!(xy.as_ref(), [0.3127, 0.3291].as_ref(), epsilon=5E-5);
+
+    // D65 Tristimulus values, using the CIE1931 standard observer by default
+    let [x, y, z] = D65.xyz(None).values();
+
+    approx::assert_ulps_eq!(x, 95.04, epsilon = 5E-3);
+    approx::assert_ulps_eq!(y, 100.0, epsilon = 5E-3);
+    approx::assert_ulps_eq!(z, 108.86, epsilon = 5E-3);
+```
+
+
+## Features
+
+This library includes a range of spectral data collections, with only a minimal set enabled by default.  
+Additional functionality can be activated using Cargo feature flags.
+
+
+<details>
+<summary><strong>Default Features</strong></summary>
+These features are enabled by default. To disable, use:
+
+```bash
+cargo add colorimetry --no-default-features
 ```
 
-## Features 
+- **`cie-illuminants`**  
+  Includes a large collection of standard illuminants (e.g., Fluorescent and LED series).  
+
+- **`supplemental-observers`**  
+  Adds the following CIE standard colorimetric observers beyond the **CIE 1931** 2º Standard Observer:
+  - **CIE 1964** 10° Standard Observer  
+  - **CIE 2015** Cone Fundamental based Standard Observers (2° & 10°)
+
+</details>
+
+<details>
+<summary><strong>Optional Features</strong></summary>
+
+- **`cie-illuminants`**
```

Comment:
`cie-illuminants` is listed both under default features and optional features. It is currently enabled by default, so it should probably be removed from the optional features here.